src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.cs

```diff
@@ -49,6 +60,57 @@ internal bool AutomaticRedirection
                 _automaticRedirection = value;
             }
         }
+        internal bool UseProxy
+        {
+            get
+            {
+                if (_proxyPolicy == CurlProxyUsePolicy.UseDefaultProxy)
+                {
+                    return false;
+                }
+                return _proxyPolicy != CurlProxyUsePolicy.DoNotUseProxy;
```

Comment:
am not really clear about the expected semantics of this property.

When  handler.UseProxy evaluates to true, which of the following is/are true?:
a.  The User has configured the handler to use Proxy property in his/her code.
b.  The subsequent http call will go via a proxy.
c. Both of the above?

I assumed it is (a) only.
If it is (b) or (c), I'll need to change the code to look up the value of environment-variable http_proxy.